_docs/quick-start.markdown

```diff
@@ -10,68 +10,44 @@ order: 10
+should see a suggestion to download the standard library:
 
-Instead of importing, you can also simply **open** a project.
+![download-stdlib](/assets/quick-start/download-stdlib-notification.png)
 
-![py-open](https://cloud.githubusercontent.com/assets/1711539/14491095/85f23ea4-017f-11e6-9809-fb4c7cbb248e.png)
 
-Select the directory with the project. Open any Rust file and configure toolchain:
+Select a directory to download the library to.
 
-![toolchain-notification](https://cloud.githubusercontent.com/assets/1711539/14825248/c6d42ed4-0be1-11e6-96f7-01c76e4cdf10.png)
+![download-stdlib-dir](/assets/quick-start/download-stdlib-directory.png)
 
-{% include h title="After your project is ready..." tag="project-ready" %}
-
-
-Wait until Cargo downloads all project dependencies. To check that everything is
-working, try **Goto Symbol** (<kbd>Ctrl+Alt+Shift+N</kbd>) and type something.
-Note that dependencies are present under external libraries. **Goto Symbol** should
-also work for items from the external crates.
-
-![go-to-symbol](https://cloud.githubusercontent.com/assets/1711539/14491412/44200bd0-0181-11e6-9587-10e4a07fa961.png)
 
-To execute Cargo tasks from within the IDE, you need to set up a [Run
-Configuration](https://www.jetbrains.com/idea/help/creating-and-editing-run-debug-configurations.html).
-**Edit configurations** (<kbd>Alt+Shift+F10</kbd>) and add a _Cargo command_ config. Be
-sure to click the green plus sign, and **not** the "Defaults" :smile:. You can also
-use <kbd>Alt+Insert</kbd> shortcut here.
+Wait until the library is downloaded and indexed.
 
-![add-run-configuration](https://cloud.githubusercontent.com/assets/1711539/14211919/33d29e60-f839-11e5-8c08-c8d09cbbf4ee.png)
 
-Fill in the name of a command and additional arguments.
-
-![edit-run-configuration](https://cloud.githubusercontent.com/assets/1711539/14211918/33ce8e56-f839-11e5-92c2-8c96bf365699.png)
-
-You should be able to compile and **Run** (<kbd>Shift+F10</kbd>) your application from the IDE now:
-
-![running](https://cloud.githubusercontent.com/assets/1711539/14211917/33cb0c54-f839-11e5-8026-d4fd7a7b44fd.png)
+{% include h title="After your project is ready..." tag="project-ready" %}
 
-You can change toolchain in the **settings** (<kbd>Ctrl+Alt+S</kbd>), under `Languages &
-Frameworks > Rust` tab. We need Cargo from at least _1.8.0_ release.
+Check that the basic functionality is working. Try **Goto Symbol**
+<kbd>Ctrl+Alt+Shift+N</kbd>, **Goto Declaration** <kbd>Ctrl+B</kbd> and
+completion <kbd>Ctrl+Space</kbd>.
 
-![settings](https://cloud.githubusercontent.com/assets/1711539/14491097/85f717d0-017f-11e6-98d6-0f60ee0e2016.png)
 
-Plugin automatically watches `Cargo.toml` for changes and communicates with
-Cargo to learn about project structure and dependencies. If you disable this
-behavior, or if you want to force a project update, you can use **Refresh Cargo
-project** action.
+{% include feature-gfy.html n="SadCourteousDobermanpinscher" %}
```

Comment:
`feature-gfy.html` wasn't meant to be used besides features page, but ok; all `.pic` styling is narrowed to be inside `.feature-group`